pttrack/models.py

```diff
@@ -232,6 +233,10 @@ class Patient(Person):
     preferred_contact_method = models.ForeignKey(ContactMethod, blank=True,
                                                  null=True)
 
+    # If the patient is in clinic and needs a workup, that is specified by needs_workup. Default value is false for all the previous patients
+
+    needs_workup = models.BooleanField(default=False)
```

Comment:
Good call. I wondered if there wasn't a good reason you were doing it that way. I'll make an issue for that.